linkis-public-enhancements/linkis-configuration/src/main/resources/mapper/common/AcrossClusterRuleMapper.xml

```diff
@@ -0,0 +1,99 @@
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+  
+<!DOCTYPE mapper PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN" "http://mybatis.org/dtd/mybatis-3-mapper.dtd" >
+
+<mapper namespace="org.apache.linkis.configuration.dao.AcrossClusterRuleMapper">
+
+    <resultMap id="acrossClusterRuleMap" type="org.apache.linkis.configuration.entity.AcrossClusterRule">
+        <id property="id" column="id" jdbcType="BIGINT"/>
+        <result property="clusterName" column="cluster_name" jdbcType="VARCHAR"/>
+        <result property="creator" column="creator" jdbcType="VARCHAR"/>
+        <result property="user" column="user" jdbcType="VARCHAR"/>
+        <result property="createTime" column="create_time" jdbcType="TIMESTAMP"/>
+        <result property="createBy" column="create_by" jdbcType="VARCHAR"/>
+        <result property="updateTime" column="update_time" jdbcType="TIMESTAMP"/>
+        <result property="updateBy" column="update_by" jdbcType="VARCHAR"/>
+        <result property="rules" column="rules" jdbcType="VARCHAR"/>
+        <result property="isValid" column="is_valid" jdbcType="VARCHAR"/>
+    </resultMap>
+
+    <sql id="AcrossClusterRule_Column_List">
+        id,cluster_name,creator,user,create_time,create_by,update_time,update_by,rules,is_valid
+    </sql>
+
+    <sql id="AcrossClusterRule_insert_Column_List">
+        cluster_name,creator,user,create_time,create_by,update_time,update_by,rules,is_valid
+    </sql>
+
+    <select id="getAcrossClusterRule" resultMap="acrossClusterRuleMap">
+        SELECT
+            acr.id, acr.cluster_name, acr.creator,
+            acr.user, acr.create_time, acr.create_by,
+            acr.update_time, acr.update_by, acr.rules
+        FROM
+            `linkis_cg_lm_across_cluster_rule` acr
+        WHERE
+            id = #{id}
+    </select>
+
+    <select id="queryAcrossClusterRuleList" resultMap="acrossClusterRuleMap">
+        select <include refid="AcrossClusterRule_Column_List"/>
+        from `linkis_cg_lm_across_cluster_rule`
+        <where>
+            <if test="user != null"> user = #{user}</if>
+            <if test="creator != null">and creator = #{creator}</if>
+            <if test="clusterName != null">and cluster_name = #{clusterName}</if>
+        </where>
+    </select>
+
+
+    <insert id="insertAcrossClusterRule" parameterType="org.apache.linkis.configuration.entity.AcrossClusterRule">
+        INSERT INTO
+            linkis_cg_lm_across_cluster_rule (<include refid="AcrossClusterRule_insert_Column_List"/>)
+        VALUES
+            (#{acrossClusterRule.clusterName},#{acrossClusterRule.creator},
+            #{acrossClusterRule.user},#{acrossClusterRule.createTime},
+            #{acrossClusterRule.createBy},#{acrossClusterRule.updateTime},
+            #{acrossClusterRule.updateBy},#{acrossClusterRule.rules},
```

Comment:
createTime  updateTime 补充默认值 
如
 ```
<choose>
      <when test="item.updateTime != null">
        #{item.updateTime}
      </when>
      <otherwise>
        now()
      </otherwise>
    </choose>
```

Reply:
在service层中处理了时间字段，未修改

Reply:
保留默认值会比较好 后续如果有其他新增接口调用 没有默认值 都需要手动设置参数值